Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -105,7 +105,7 @@ inline void ComputeStrokeStyle(ID2D1DeviceContext* deviceContext) {
         deviceContext->GetFactory(&factory);
 
         std::vector<float> adjustedDashes(dashes.size());
-        std::transform(dashes.cbegin(), dashes.cend(), adjustedDashes.begin(), [this](const float& f) -> float { return f / lineWidth; });
+        std::transform(dashes.cbegin(), dashes.cend(), adjustedDashes.begin(), [this](const CGFloat& f) -> float { return f / lineWidth; });
```

Comment:
In a future world where we differentiate CGFloat based on some platform-specific thing, yes.

---
In reply to: [85250462](https://github.com/Microsoft/WinObjC/pull/1240#discussion_r85250462) [](ancestors = 85250462)